dbms/src/DataStreams/materializeBlock.cpp

```diff
@@ -7,7 +7,7 @@ namespace DB
 Block materializeBlock(const Block & block)
 {
     if (!block)
-        return block;
+        return Block();
```

Comment:
Nothing wrong was here. But this is Ok also.

Reply:
Yep, I also dont see any problem as the previous code from my point of view was valid, however, it emitted warning